.pipelines/templates/rp-dev/rp-dev-vars.yml

```diff
@@ -0,0 +1,8 @@
+variables:
+  AZURE_UNIQUE_PREFIX: zzz # dummy value
+  NO_CACHE: false
+  AZURE_EXTENSION_DEV_SOURCES: "$(pwd)/python"
```

Comment:
Is there a reason for us to use the ARO-RP dev extension for deploying an int-like environment, or can we get away with production `az aro`? I don't think any commands for setting up an int-like rely on the `az aro` commandlet?

Reply:
I agree, it can be discarded. I hope we don't miss something that happens automatically.

Having said that this automation doesn't include the OCP cluster creation, but if/when it does, then I believe the above extension will be required as it is used in https://github.com/Azure/ARO-RP/blob/master/docs/deploy-full-rp-service-in-dev.md#deploy-a-cluster step 8 (`az aro create \`).